layouts/partials/contact.html

```diff
@@ -7,15 +7,15 @@
 
                 {{ .Content }}
 
-                {{ if isset .Site.Params "email" }}
+                {{ if isset .Site.Params "endpoint" }}
 
                 <div class="heading">
                   <h3>{{ i18n "contactForm" }}</h3>
                 </div>
 
                 <div id="contact-message"></div>
 
-                <form {{ with .Site.Params.contact_form_ajax }}class="contact-form-ajax"{{ else }}{{ end }} method="post" action="https://formspree.io/{{ .Site.Params.email }}">
+                <form {{ with .Site.Params.contact_form_ajax }}class="contact-form-ajax"{{ else }}{{ end }} method="post" action="https://formspree.io/{{ .Site.Params.endpoint }}">
```

Comment:
I prefer action have all the URL like: `https://formspree.io/test@dada.com`

Reply:
OK.
But I'll change a variable name to 'formspree_action' because a variable name in config.toml is "formspree_action" from your review.